commands/reputation.go

```diff
@@ -0,0 +1,237 @@
+package commands
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/olekukonko/tablewriter"
+	"go.mozilla.org/iprepd"
+	cli "gopkg.in/urfave/cli.v1"
+)
+
+// ReputationCmd is the CLI command object for reputation operations
+var ReputationCmd = cli.Command{
+	Name:    "reputation",
+	Aliases: []string{"r"},
+	Usage:   "reputation entry related commands",
+	Subcommands: []cli.Command{
+		{
+			Name:  "list",
+			Usage: "list all available reputation entries",
+			Flags: []cli.Flag{
+				jsonFlag,
+			},
+			Action: reputationListHandler,
+		},
+		{
+			Name:  "get",
+			Usage: "get the entry for a given object",
+			Flags: []cli.Flag{
+				asMandatory(objectFlag),
+				withDefault(typeFlag, "ip"),
+				jsonFlag,
+			},
+			Before: reputationGetValidator,
+			Action: reputationGetHandler,
+		},
+		{
+			Name:  "set",
+			Usage: "update the entry for a given object",
```

Comment:
Let's match up the command name and the usage description, as in use `set` or `update` but not both.